b2sdk/_internal/file_version.py

```diff
@@ -14,14 +14,15 @@
 from copy import deepcopy
 from typing import TYPE_CHECKING, Any
 
+from b2sdk._internal.progress import AbstractProgressListener
+from b2sdk._internal.utils import Sha1HexDigest, b2_url_decode
+from b2sdk._internal.utils.http_date import parse_http_date
+from b2sdk._internal.utils.range_ import Range
+
 from .encryption.setting import EncryptionSetting, EncryptionSettingFactory
 from .file_lock import NO_RETENTION_FILE_SETTING, FileRetentionSetting, LegalHold
 from .http_constants import FILE_INFO_HEADER_PREFIX_LOWER, LARGE_FILE_SHA1, SRC_LAST_MODIFIED_MILLIS
-from .progress import AbstractProgressListener
 from .replication.types import ReplicationStatus
-from .utils import Sha1HexDigest, b2_url_decode
-from .utils.http_date import parse_http_date
-from .utils.range_ import Range
 
```

Comment:
Why are some relative imports converted to absolute imports? In my understanding, the imports here did not need to change.